Automattic/wp-calypso
# Update/refactor design first flow

Related to #86432

## Proposed Changes
The `design-first` flow is using the following tasks:

| TASK | MIGRATED? |
|--------|--------|
| first_post_published | THIS PR |
| setup_blog | Previously |
| domain_upsell | Previously |
| plan_completed | THIS PR |
| blog_launched | THIS PR | 

## Testing Instructions
* Start the flow ` start-writing`  (setup/start-writing)
* Go to the launchpad page (setup/start-writing/launchpad)
* Check if  there are messages on devtools to all task above
* Use the launchpad (Set values, Edit values, Check if the values are marked when completed)
* Finish the flow.

Requires the feature flags:
* "launchpad/new-task-definition-parser"
* "launchpad/new-task-definition-parser/start-writing"

The flags can be enabled appending  the `?flags=` flag on the url (Please always check if the flags= param is still on the url when the system redirect back the user from some "no launchpad" page ).

## Pre-merge Checklist
